config/checkstyle/suppressions.xml

```diff
@@ -27,4 +27,6 @@
 
 <suppressions>
     <suppress files="[/\\]build[/\\]generated[/\\]" checks=".*"/>
+    <!-- TODO: fix integration test formatting -->
+    <suppress files="[/\\]integration-tests[/\\]src[/\\]test[/\\]main[/\\]" checks=".*"/>
```

Comment:
I'm guessing we currently haven't covered integration-tests? We should perhaps try to fix that, first? Or leave as is?

Reply:
For now I'd leave the exclusion. Checkstyle reports an avalanche of 1500 errors. I'd say we can fix this when we migrate step by step to to fineract-client usage.

Reply:
OK. Let's follow-up on that in FINERACT-1246. I'll leave this review comment open.